Dockerfile_clearlinux

```diff
@@ -1,15 +1,112 @@
+
+RUN make --jobs=$(nproc --all)
+
+
+FROM clearlinux:latest as clear-pkg
+
+RUN swupd update --no-boot-update \
+    && swupd bundle-add wget unzip lftp package-utils cpio \
+    && rm -rf /var/lib/swupd
+
+RUN if [ ! -e "/usr/lib64/libzstd.so.1"  ]; then ln -sf /usr/lib64/haswell/libzstd.so.1.4.5 /usr/lib64/libzstd.so.1 ; fi
+
+WORKDIR /pkg
+
+RUN mkdir -p rpms/runtime &&  source /etc/os-release \
+    && cd rpms/runtime \
+    && lftp -c "o https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/; mget clinfo-bin*.rpm intel-compute-runtime-data*.rpm intel-compute-runtime-lib*.rpm intel-gmmlib-lib*.rpm intel-graphics-compiler-lib*.rpm libva-lib*.rpm ocl-icd-lib*.rpm opencl-clang-lib*.rpm"
+
+RUN mkdir -p out/runtime \
+    && for p in `ls rpms/runtime/*.rpm`; do rpm2cpio $p | (cd out/runtime/; cpio -i -d -u --quiet); done \
+    && (cd out/runtime ; echo "OpenCL Runtime files:" ; find . -type f -or -type l)
+
+RUN mkdir -p rpms/tf && pwd && source /etc/os-release \
+    && cd rpms/tf \
+    && lftp -c "o https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/; mget tensorflow-2.0.1-98.x86_64.rpm numpy-*  openblas-* Werkzeug* protobuf*  absl-py* wrapt* gast* astor* termcolor* Keras_Applications* Keras_Preprocessing*"
+
+RUN mkdir -p out/tf \
+    && for p in `ls rpms/tf/*.rpm`; do rpm2cpio $p | (cd out/tf/; cpio -i -d -u --quiet); done \
+    && (cd out/tf ; echo "TensorFlow files:" ; find . -type f -or -type l) \
+    && du -shc out/*
+
+
 FROM clearlinux
 
-RUN swupd update && swupd bundle-add computer-vision-basic && swupd clean
-# OpenVino R3 version
+RUN swupd update && \
+    swupd bundle-add \
+    python3-basic \
+    devpkg-libusb \
+    && rm -rf /var/lib/swupd
 
-COPY start_server.sh setup.py requirements_clearlinux.txt version /ie-serving-py/
-COPY ie_serving /ie-serving-py/ie_serving
+COPY --from=build /openvino/inference-engine/temp/tbb/lib /usr/local/lib/inference-engine
+COPY --from=build /openvino/bin/intel64/Release/lib /usr/local/lib/inference-engine
+COPY --from=clear-pkg /pkg/out/runtime/ /
+COPY --from=clear-pkg /pkg/out/tf/ /
 
 WORKDIR /ie-serving-py
 
-RUN pip --no-cache-dir install -r requirements_clearlinux.txt
+COPY start_server.sh setup.py version requirements_clearlinux.txt /ie-serving-py/
+COPY --from=build /tmp/pip/tensorflow_serving_api-*.whl .
+
+RUN pip3 install --no-cache-dir tensorflow_serving_api-*.whl && \
+    pip3 --no-cache-dir install -r requirements_clearlinux.txt && \
+    rm tensorflow_serving_api-*.whl
+
+COPY ie_serving /ie-serving-py/ie_serving
+
+RUN pip3 --no-cache-dir install .
 
-RUN pip install .
+ENV LD_LIBRARY_PATH=/usr/local/lib/inference-engine:/usr/local/lib
+ENV PYTHONPATH=/usr/local/lib:/usr/local/lib/inference-engine/python_api/python3.8:/usr/local/lib/inference-engine/python_api/python3.8/openvino/inference_engine
```

Comment:
Somewhere in OpenVINO toolkit should be an environment file with PYTHONPATH and some other envs. Maybe it would be better to use that file instead _believing_ it should be fine? Or are we certain that it will work all the time?

@dtrawins I remember you were talking about it some time ago, can you evaluate?